### PR TITLE
update dotnet versions

### PR DESCRIPTION
**Pull request by kasperk81**
adding new versions without hiding the old ones. when we will reach about 10+ versions, then we can hide the oldest ones (.hidden=true) to keep the permalinks. right now there is no point in hiding few months old versions as the dropdown is pretty much empty

### CODE REVIEW

File: etc/config/csharp.amazon.properties [hunk 1]
```diff
@@ -7,16 +7,28 @@ defaultCompiler=dotnet700csharp
 group.csharp.compilers=dotnettrunkcsharp:dotnet700csharp:dotnet6011csharp
 
 compiler.dotnet6011csharp.exe=/opt/compiler-explorer/dotnet-v6.0.11/.dotnet/dotnet
-compiler.dotnet6011csharp.name=.NET 6.0.403
+compiler.dotnet6011csharp.name=.NET 6.0.110
 compiler.dotnet6011csharp.clrDir=/opt/compiler-explorer/dotnet-v6.0.11
 compiler.dotnet6011csharp.buildConfig=Release
 compiler.dotnet6011csharp.langVersion=latest
 
-compiler.dotnet700csharp.exe=/opt/compiler-explorer/dotnet-v7.0.0/.dotnet/dotnet
```
Comment (RubenRBS) [inline etc/config/csharp.amazon.properties:15]: Compiler ids should always be present if they have been deployes live, else old links will break

**Review by RubenRBS — CHANGES_REQUESTED** [review]

File: etc/config/csharp.amazon.properties (hunk 1 above)
Comment (kasperk81) [reply etc/config/csharp.amazon.properties:15]: this was "wrong" in first place. starting from this point, we will freeze them.

Comment (kasperk81) [reply etc/config/csharp.amazon.properties:15]: added as alias

**Comment by kasperk81** [comment]
@mattgodbolt

**Review by mattgodbolt — APPROVED** [review]
I'm ok with this. Sorry for the delay.

**Pull request merged**